modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -597,7 +616,7 @@ When users log in to the domain, the client has received the necessary Kerberos
 
 image::enterprise/authentication/kerberos/kerberos-login-screenshot.png[Alternative Windows Domain Login, width=300]
 
-Compared to a standard login using a user name and password, the user clicks on the btn:[Windows Domain Login] button. With this button, the browser takes the Kerberos ticket from the client and uses it for the ownCloud login process. Kerberos tickets are further used to access file resources via the xref:enterprise/external_storage/windows-network-drive_configuration.adoc[Windows Network Drive (WND)] app without the need of a user authenticating again.
+Compared to a standard login using a user name and password, the user clicks on the btn:[Windows Domain Login] button. With this button, ownCloud requests a new ticket on behalf of the Windows user logged in, which is technically a constrained delegation. With this new ticket, all other ownCloud apps that provide Kerberos authentications can then use this ticket, like the xref:enterprise/external_storage/windows-network-drive_configuration.adoc[Windows Network Drive (WND)] app.
```

Comment:
This is not correct. The delegation doesn't happen for the login with the Kerberos App. Delegation is only done in the WND App (and also not in `all other ownCloud apps`. The two parts are completely disjunct. So either we remove the WND- related sentence here completely to avoid confusion or make clear that this is a two-step process.

```suggestion
Compared to a standard login using a user name and password, the user clicks on the btn:[Windows Domain Login] button. With this button, ownCloud requests a new ticket on behalf of the Windows user logged in, which is technically a constrained delegation. With this button, the browser takes the Kerberos ticket from the client and uses it for the ownCloud login process. If the WND App is installed as well the ownCloud service user might later on request additional Kerberos ticket on behalf of the Windows user logged in, which is technically a constrained delegation.
```